loader-tests/package.json

```diff
@@ -3,12 +3,14 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "check-opml": "tsx check-opml.ts"
+    "check-opml": "tsx check-opml.ts",
+    "check-feeds": "tsx check-feeds.ts feeds.yml"
```

Comment:
What do you think of renaming it to `test-loaders` since we are not testing the feeds (websites).